madlibs/src/addCommas.js

```diff
@@ -1,3 +1,11 @@
-function addCommas() {}
+function addCommas(n) {	
+    if (n === parseFloat(n)) {
+    return parseFloat(n).toLocaleString();
+}
+if (n === parseInt(n)) {
+    return n.toLocaleString();
+}
+return NaN;
+}

```

Comment:
JS isn't whitespace-sensitive but best to keep your code properly formatted
```suggestion
function addCommas(n) {	
    if (n === parseFloat(n)) {
        return parseFloat(n).toLocaleString();
    }
    if (n === parseInt(n)) {
        return n.toLocaleString();
    }
    return NaN;
}

```

Reply:
Also, this gives you the correct answer but the assessment is really about you creating an algorithm for adding in commas yourself.